duckduckgo/iOS
# Prevent the content from expanding when scrolled

As reported on issue #730 the previous page is blank after swiping back but I have not encountered that issue.



Task/Issue URL: https://github.com/duckduckgo/iOS/issues/730
Tech Design URL:
CC:

**Description**:
The web view content inset was set to prevent the resizing.

**Steps to test this PR**:
1. Visit a page like https://nos.nl
2. Scroll down to hide the scrollbars
3. The images or content should not expand
4. Swipe back
5. The previous page should appear



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
